charts/incubator/notifiarr/values.yaml

```
@@ -1,7 +1,7 @@
 image:
   repository: tccr.io/truecharts/notifiarr
   pullPolicy: IfNotPresent
-  tag: latest@sha256:913c71f5e325dd9af53c9e96f3d5d7d1b2d058a41f8253489320a4c69fbb0a44
+  tag: latest@sha256:b7c07ad5e1db64bba686d548bf36a426f37cea1f90f7707a8c3221ab015abd9d
```

Chart.yaml:
set appVersion to "0.6.0"
```suggestion
  tag: v0.6.0@sha256:200107f99dd63811270da14db2228471381e7531c64dd9c6162afeabb08b7576
```